### ISSUE TITLE
Missing dependency error in new project

### ISSUE THREAD

**Issue by chrisdc** [nuxt/bridge#715]
### Environment

Nuxi 3.1.2                                                                                                                                                     11:52:23
                                                                                                                                                               11:52:24
RootDir: /home/chrisdc/Documents/nuxt-bridge-error-reporduction                                                                                                11:52:24
Nuxt project info:                                                                                                                                             11:52:24

------------------------------
- Operating System: `Linux`
- Node Version:     `v16.13.2`
- Nuxt Version:     `2.16.2`
- Nitro Version:    `1.0.0`
- Package Manager:  `npm@9.6.1`
- Builder:          `webpack`
- User Config:      `ssr`, `target`, `head`, `css`, `plugins`, `components`, `buildModules`, `modules`, `build`, `serverHandlers`, `devServerHandlers`, `devServer`, `typescript`, `bridge`
- Runtime Modules:  `-`
- Build Modules:    `()`, `@nuxt/bridge@3.0.0-27928371.c30fa0f`
------------------------------

### Reproduction

The [starter template on Stackblitz](https://stackblitz.com/github/nuxt/starter/tree/v2-bridge) may be used as a reprorduction without any modification, but the steps below may be followed to reproduce the issue locally. See also https://github.com/chrisdc/nuxt-bridge-error-reporduction

1. Run `npx create-nuxt-app bridge-test`
2. Select the following settings
    - Project name: use the default
    - Programming language: JavaScript
    - Package manager: Npm
    - UI framework: None
    - Template engine: HTML
    - Nuxt.js modules: none
    - Linting tools: none
    - Testing framework: none
    - Rendering mode: Single page app
    - Deployment target: Static
    - Development tools: none
    - Version contol system: Git
3. Navigate to the new folder
4. Following the upgrade instructions listed on this repository update to the most recent version of nuxt 2 (2.16.2) then install Bridge: `npm install -D @nuxt/bridge@npm:@nuxt/bridge-edge`
5. Update nuxt.config.js to use `defineNuxtConfig` and update the npm commands to use nuxi.
6. Run `npm run dev`

### Describe the bug

Attempting to start the dev server produces a number of warnings and errors as reported under the logs section. Opening localhost:3000 in the browser renders a blank page with the same error reported in the console.

### Logs

```shell
✖ Client
  Compiled with some errors in 7.50s

 WARN  Compiled with 10 warnings                                                                                                               friendly-errors 11:55:43


 WARN  in ./components/NuxtLogo.vue                                                                                                            friendly-errors 11:55:43

"export 'render' was not found in './NuxtLogo.vue?vue&type=template&id=32dab8a9&'                                                              friendly-errors 11:55:43
                                                                                                                                               friendly-errors 11:55:43

 WARN  in ./components/Tutorial.vue                                                                                                            friendly-errors 11:55:43

"export 'render' was not found in './Tutorial.vue?vue&type=template&id=63aed68e&'                                                              friendly-errors 11:55:43
                                                                                                                                               friendly-errors 11:55:43

 WARN  in ./.nuxt/layouts/default.vue                                                                                                          friendly-errors 11:55:43

"export 'render' was not found in './default.vue?vue&type=template&id=cac06082&'                                                               friendly-errors 11:55:43
                                                                                                                                               friendly-errors 11:55:43

 WARN  in ./pages/index.vue                                                                                                                    friendly-errors 11:55:43

"export 'render' was not found in './index.vue?vue&type=template&id=2a183b29&'                                                                 friendly-errors 11:55:43
                                                                                                                                               friendly-errors 11:55:43

 WARN  in ./.nuxt/components/nuxt-error.vue                                                                                                    friendly-errors 11:55:43

"export 'render' was not found in './nuxt-error.vue?vue&type=template&id=74e3df5b&'                                                            friendly-errors 11:55:43
                                                                                                                                               friendly-errors 11:55:43

 WARN  in ./components/NuxtLogo.vue                                                                                                            friendly-errors 11:55:43

"export 'staticRenderFns' was not found in './NuxtLogo.vue?vue&type=template&id=32dab8a9&'                                                     friendly-errors 11:55:43
                                                                                                                                               friendly-errors 11:55:43

 WARN  in ./components/Tutorial.vue                                                                                                            friendly-errors 11:55:43

"export 'staticRenderFns' was not found in './Tutorial.vue?vue&type=template&id=63aed68e&'                                                     friendly-errors 11:55:43
                                                                                                                                               friendly-errors 11:55:43

 WARN  in ./.nuxt/layouts/default.vue                                                                                                          friendly-errors 11:55:43

"export 'staticRenderFns' was not found in './default.vue?vue&type=template&id=cac06082&'                                                      friendly-errors 11:55:43
                                                                                                                                               friendly-errors 11:55:43

 WARN  in ./pages/index.vue                                                                                                                    friendly-errors 11:55:43

"export 'staticRenderFns' was not found in './index.vue?vue&type=template&id=2a183b29&'                                                        friendly-errors 11:55:43
                                                                                                                                               friendly-errors 11:55:43

 WARN  in ./.nuxt/components/nuxt-error.vue                                                                                                    friendly-errors 11:55:43

"export 'staticRenderFns' was not found in './nuxt-error.vue?vue&type=template&id=74e3df5b&'                                                   friendly-errors 11:55:43
                                                                                                                                               friendly-errors 11:55:43

 ERROR  Failed to compile with 1 errors                                                                                                        friendly-errors 11:55:43

This dependency was not found:                                                                                                                 friendly-errors 11:55:43
                                                                                                                                               friendly-errors 11:55:43
* -!../../node_modules/unplugin/dist/webpack/loaders/transform.js??nuxt:imports-transform!../../node_modules/babel-loader/lib/index.js??ref--4-0!../../node_modules/vue-loader/lib/loaders/templateLoader.js??ref--9!../../node_modules/vue-loader/lib/index.js??vue-loader-options!./nuxt-error.vue?vue&type=template&id=74e3df5b& in ./.nuxt/components/nuxt-error.vue?vue&type=template&id=74e3df5b&
                                                                                                                                               friendly-errors 11:55:43
To install it, you can run: npm install --save -!../../node_modules/unplugin/dist/webpack/loaders/transform.js??nuxt:imports-transform!../../node_modules/babel-loader/lib/index.js??ref--4-0!../../node_modules/vue-loader/lib/loaders/templateLoader.js??ref--9!../../node_modules/vue-loader/lib/index.js??vue-loader-options!./nuxt-error.vue?vue&type=template&id=74e3df5b&
```

**Comment by antlionguard** [comment]
This project not good for new projects. Only people using the nuxt 2 to 3 migration process are using this project. I was wondering why you opened a new "nuxt bridge" project.

**Comment by chrisdc** [comment]
> This project not good for new projects. Only people using the nuxt 2 to 3 migration process are using this project. I was wondering why you opened a new "nuxt bridge" project.

Hi @antlionguard thanks for the reply. Sorry, perhaps I should have mentioned this in the PR, but I am actually trying to convert an existing (and quite large) Nuxt 2 project and running into this issue. I can't share the code from that project publicly, so I am using a new project to provide a minimal reproduction.

**Comment by MartinX3** [comment]
I also get this problem with my nuxt bridge project out of the sudden.
So it does not only happen with new projects.

**Comment by chrisdc** [comment]
I don't fully understand the interactions occurring here yet, but it appears to work if I force npm to use unplugin 1.1.0

```
"overrides": {
    "unplugin": "1.1.0"
}
```

The subsequent versions: 1.2.0 and 1.3.0 have both been released in the last few days, so I guess there's some kind of compatibility issue with these version.

**Comment by MartinX3** [comment]
@chrisdc you're right
That's the cause

**Comment by wattanx** [comment]
It seems to have stopped reproducing.
Maybe it has been fixed.

**Comment by chrisdc** [comment]
@wattanx I can't reproduce the error any more either. It appears that it may have been resolved in unplugin 1.3.1. See https://github.com/unjs/unplugin/issues/286
